pl.js

```diff
@@ -812,7 +812,7 @@ social.market.fee.gift Brak podatku
 social.market.gift Wyślij KR
 social.market.gift.msg Zostaw wiadomość
 respawn.auto Automatyczne odrodzenie za {0}
-custom.host.ranked Nie możesz hostować gier, gdy grasz w trybie rankowanym.
+custom.host.ranked Nie możesz hostować gier, gdy grasz w trybie rankingowym.
```

Comment:
For the sake of consistency, I think "Nie możesz tworzyć rozgrywek..." or "Nie możesz utworzyć rozgrywki..." or maybe "Nie można tworzyć rozgrywek, grając w trybie rankingowym"

Reply:
Missed this one